yarn.lock

```diff
@@ -4554,6 +4571,14 @@ json-rpc-engine@^5.1.3:
     promise-to-callback "^1.0.0"
     safe-event-emitter "^1.0.1"
 
+json-rpc-engine@^5.3.0:
```

Comment:
Nit: This could be used for the `json-rpc-engine@^5.1.3` dependency as well, so we'd only have one copy.

Reply:
The `web3-provider-engine` PR will take care of this!